openedx/features/offline_mode/assets_management.py

```diff
@@ -176,3 +176,35 @@ def save_external_file(temp_dir, link, filename):
         create_subdirectories_for_asset(file_path)
         with open(file_path, 'wb') as file:
             file.write(response.content)
+
+
+def get_offline_course_folder_path(course_id):
+    """
+    Generates the base storage path for the given course.
+
+    Args:
+        course_id (str): The course ID for which to generate the storage path.
+    Returns:
+        str: The constructed base storage path.
+    """
+    return f'offline_content/{course_id}/'
```

Comment:
What do you think about moving this to the settings as a formatted string where the course ID will be substituted?